src/ImmutableX.ts

```diff
@@ -350,6 +356,92 @@ export class ImmutableX {
       });
   }
 
+  /**
+   * Get a list of metadata refreshes
+   * @param ethSigner - the L1 signer
+   * @param collectionAddress - the Collection contract address
+   * @param pageSize - the page size of the result
+   * @param cursor - the cursor
+   * @returns a promise that resolves with the requested metadata refreshes
+   * @throws IMXError
+   */
+  public getAListOfMetadataRefreshes(
```

Comment:
Strange name, should this be `listMetadataRefreshes`?

Reply:
Renamed
getAListOfMetadataRefreshes -> listMetadataRefreshes
requestAMetadataRefresh -> requestMetadataRefresh
[ec318f3](https://github.com/immutable/imx-core-sdk/pull/175/commits/ec318f34ce13aad114ca275347613d3f015d546c)